src/io/flutter/view/FlutterView.java

```diff
@@ -150,6 +192,13 @@ private void updateIcon() {
       else {
         toolWindow.setIcon(ExecutionUtil.getLiveIndicator(FlutterIcons.Flutter_13));
       }
+
+      System.out.println("App changed... " + (app == null));
```

Comment:
Remove `System.out`?

Reply:
Alternatively, consider using a `Logger` or putting this debugging output behind a flag?

Reply:
Removed. That was left over debugging code.